### PR TITLE
fix(adiru): wind velocity calculation

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

Plane magically measuring the correct wind velocity while firmly planted on the runway got you down? Has this PR got the solution for you!

(also fixes nomenclature... velocity = vector, not scalar)

## References

FBW-34-02

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/systems/systems/src/navigation/adirs.rs [hunk 1]
```diff
@@ -1195,9 +1217,108 @@ impl InertialReference {
...
+        // The IR does not compute the wind if the TAS is less than 100 knots or NCD
+        let true_airspeed_above_minimum_threshold = true_airspeed_source
+            .true_airspeed()
+            .is_normal_operation()
+            && true_airspeed_source.true_airspeed().value()
+                >= Velocity::new::<knot>(Self::MINIMUM_TRUE_AIRSPEED_FOR_WIND_DETERMINATION_KNOTS);
+
+        let ssm = if no_transmission {
+            SignStatus::FailureWarning // TODO should be no transmission
+        } else if true_airspeed_above_minimum_threshold {
```
Comment (tracernz) [inline src/systems/systems/src/navigation/adirs.rs:1244]: Damn... refactor took out the NAV mode condition here... TODO 😬

File: src/systems/systems/src/navigation/adirs.rs [hunk 2]
```diff
@@ -1195,9 +1217,108 @@ impl InertialReference {
...
+
+            while wind_direction < 0. {
+                wind_direction += 360.;
+            }
+            wind_direction += 180.;
+            while wind_direction >= 360. {
+                wind_direction -= 360.;
+            }
+        } else {
+            self.wind_velocity.reset(Vector2::new(0., 0.));
```
Comment (crocket63) [inline src/systems/systems/src/navigation/adirs.rs:1285]: Vector2::default() should work here

**Review by lukecologne — APPROVED** [review]

**Comment by lukecologne** [comment]
Calculation looks good, just need to run cargo fmt

File: src/systems/systems/src/navigation/adirs.rs [hunk 3]
```diff
@@ -1195,9 +1198,111 @@ impl InertialReference {
...
+            wind_speed = self
+                .wind_velocity
+                .output()
+                .magnitude()
+                .clamp(0., 255.)
+                .round();
+
+            let direction =
+                Rotation2::rotation_between(&self.wind_velocity.output(), &Vector2::y());
+            //wind_direction = (Radians(direction.angle()).normalized() + Degrees(180.)).normalized().in_degrees();
```
Comment (beheh) [inline src/systems/systems/src/navigation/adirs.rs:1267]:
```suggestion
```

**Comment by alexr4339** [comment]
Quality Assurance Tester Report

Discord          : Alex R#6663
Object of testing: #PR7602
Tier of Testing  : 1
Date             : 02/12/2022

Testing Process: 
Loaded aircraft on runway with live weather, took off and climbed until 3000 feet (1800ft AGL) and checked the wind data displaying on the ND. On the takeoff roll it showed more or less the runway track (depending on how well I kept it on the centerline) and a windspeed of 0 or 1 knots. This could be excpected as I was on the ground. After rotation it showed a wind data of 347/5. The METAR for the airfield at the time showed a wind of 340/4. As I climbed and reached 3000, it showed 040/6. To check the data I took off with the DA62 and G1000 NXi and climbed until 3000ft. The Nxi showed a wind speed of 350/6. However I must say that real world waether data would indicate 030/5 at 3000 feet. For testing purposes I climbed on runway track up to FL150, here it indicated 155/8 which also correlates to real world weather data which forecasts wind of 150/10 at FL150. The weather app I used for real world weather data is actually used for flying in real life and gives pilots almost any weather information at multiple altitudes. I came to the conclusion that the wind calculatuion of the NXi is likely wrong, whereas the A32NX's wind indication correlates with real world data and seems to be correct.

Negatives:       
N/A

Testing Results: 
Passed

**Pull request merged**